reference/test/distributed/matrix_kernels.cpp

```diff
@@ -436,27 +416,27 @@ TYPED_TEST(Matrix, BuildsLocalNonLocalMixedWithColPartition)
             this->ref, col_mapping, num_parts);
 
     this->validate(gko::dim<2>{6, 6}, partition, col_partition,
+        // clang-format off
                    {2, 3, 3, 0, 5, 1, 4, 2, 3, 2, 0, 0, 1, 1, 4, 4},
-                   {0, 0, 1, 5, 4, 2, 2, 3, 2, 4, 1, 2, 4, 5, 0, 5},
+                   { 0,  0,  1,  5,  4,  2,  2, 3, 2, 4, 1, 2, 4, 5, 0, 5},
```

Comment:
nit: old formatting was better

Reply:
for me it is helpful to have the column idxs and the values aligned. I think all of these mappings are based on a drawing I used, and the alignment helped me to prevent mixing things up.

Reply:
Especially if the row and column partitions are different and both are non-trivial.